packages/fast-usdc/src/exos/transaction-feed.js

```diff
@@ -73,45 +75,94 @@ export const prepareTransactionFeedKit = (zone, zcf) => {
          * @returns {Promise<Invitation<OperatorKit>>}
          */
         makeOperatorInvitation(operatorId) {
-          const { admin } = this.facets;
+          const { creator } = this.facets;
           trace('makeOperatorInvitation', operatorId);
 
           return zcf.makeInvitation(
             /** @type {OfferHandler<OperatorKit>} */
             seat => {
               seat.exit();
-              return admin.initOperator(operatorId);
+              return creator.initOperator(operatorId);
             },
             INVITATION_MAKERS_DESC,
           );
         },
+        /** @param {string} operatorId */
+        async initOperator(operatorId) {
+          const { operators, pending } = this.state;
+          trace('initOperator', operatorId);
+
+          const operatorKit = makeOperatorKit(
+            operatorId,
+            this.facets.operatorPowers,
+          );
+          operators.init(operatorId, operatorKit);
+          pending.init(operatorId, makeScalarBigMapStore('pending evidence'));
```

Comment:
not `zone.detached().mapStore(...)`?

If we use `makeScalarBigMapStore`, do we need an explicit "make it durable" arg?

Reply:
thx, I wanted something from zone but forgot about detached()